Atomic/storage.py

```diff
@@ -19,6 +19,30 @@
 except ImportError:
     from atomic import Atomic
 
+def modify_sh_var_in_text(text, var, modifier, default=""):
+    pattern = '^[ \t]*%s[ \t]*=[ \t]*"(.*)"[ \t]*$' % re.escape(var)
+    found = [ False ]
+    def sub(match):
```

Comment:
Because `nonlocal` is only in Python 3, I think.  Is there a better idiom? Should I use `sub.found`?